.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 build
 coverage
 .idea
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode/
```

Comment:
Is updating the skin config submodule (below) deliberate? I don't see any changes on the diffs for that

Reply:
It's just automatically updated due to latest master version is newer than the one is referred from .gitmodules. But particular files being pulled via .gitmodules settings are not changed, so this update actually makes no sense. I can roll it back if necessary.

Reply:
Oh ok no problem. If it's the latest from master then it's fine